<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RgbColorParametersControlWidget</class>
 <widget class="QWidget" name="RgbColorParametersControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RgbColorParameters</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label_0">
     <property name="text">
      <string>R</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="rSpinBox">
     <property name="toolTip">
      <string>R</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>128</number>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QSpinBox" name="gSpinBox">
     <property name="toolTip">
      <string>G</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>128</number>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_1">
     <property name="text">
      <string>G</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QSpinBox" name="bSpinBox">
     <property name="toolTip">
      <string>B</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>128</number>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>B</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>1</number>
      </property>
      <property name="spacing">
       <number>1</number>
      </property>
      <item row="0" column="0">
       <widget class="ViAreaWidget" name="pickerWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="3" margin="3"/>
 <customwidgets>
  <customwidget>
   <class>ViAreaWidget</class>
   <extends>QWidget</extends>
   <header>viAreaWidget.h</header>
   <container>1</container>
   <slots>
    <signal>askParentRepaint(QPaintEvent*,QWidget*)</signal>
    <signal>notifyParentMouseRelease(QMouseEvent*)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>rSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RgbColorParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RgbColorParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>449</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RgbColorParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>674</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pickerWidget</sender>
   <signal>askParentRepaint(QPaintEvent*,QWidget*)</signal>
   <receiver>RgbColorParametersControlWidget</receiver>
   <slot>repaintPicker(QPaintEvent*,QWidget*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>783</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pickerWidget</sender>
   <signal>notifyParentMouseRelease(QMouseEvent*)</signal>
   <receiver>RgbColorParametersControlWidget</receiver>
   <slot>pickerClicked(QMouseEvent*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>783</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pickerWidget</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>721</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pickerWidget</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>730</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pickerWidget</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>596</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>762</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeParameters()</slot>
  <slot>activatePicker()</slot>
  <slot>repaintPicker(QPaintEvent*,QWidget*)</slot>
  <slot>pickerClicked(QMouseEvent*)</slot>
 </slots>
</ui>
